### ISSUE TITLE
Error when defining a class attribute in a control block: "cannot juxtapose string literal".

### ISSUE THREAD

**Issue by fypc** [MommaWatasu/OteraEngine.jl#37]
Hi,

First of all, thank you for sharing this work. I'm trying to use a `for` loop in a control block to display a table. Everything works fine until I define a class attribute. For example:

```html
{% for i in contacts %}
  <tr>
    <th class="font-medium"> {{ i["first"] }} </th>
    <td>{{ i["last"] }}</td>
    <td>{{ i["phone"] }}</td>
    <td>{{ i["email"] }}</td>
  </tr>
{% end %}
```
In this case, I got the following error message:

```
│  <tr>
│    <th class="font-medium"> ";txt *= htmlesc(string(i["first"]));txt *= " </th>
│ #             └ ── cannot juxtapose string literal
```

Hope you can give me a hint!

**Comment by MommaWatasu** [comment]
Thank you for using my package.
I think you should `htmlesc` to escape special characters inside HTML.
If it still doesn't work, please wait for a while.

**Comment by fypc** [comment]
Thanks for your answer. Unfortunately, using `htmlesc` doesn't seem to work. In the meantime, there's also the option of using single quotes.

**Comment by MommaWatasu** [comment]
I fixed the bug and released new version v0.5.7
Thanks again for your report.
